setup/environment-linux.yml

```diff
@@ -137,9 +137,9 @@ dependencies:
     - toolz==0.9.0
     - twisted==18.9.0
     - txaio==18.8.1
-    - typing-extensions==3.7.2
+    - typing-extensions==3.7.4.1
     - virtualenv==16.7.6
-    - web3==4.8.3
+    - web3==5.5.1
```

Comment:
this is a major version bump. I think there might be deprecated code we might need to fix? Do we need to do this now?

Reply:
I think fixed all now?